duqtools/config/_runs.py

```diff
@@ -4,9 +4,10 @@
 
 from pydantic import DirectoryPath, Field
 
-from duqtools.ids import IDSOperation, ImasLocation
+from duqtools.ids._location import ImasLocation
```

Comment:
It might also be nice to use local imports for these cases:

```suggestion
from ..ids._location import ImasLocation
```